### PR TITLE
docs: add explanation of ip whitelist

### PR DESCRIPTION
**Pull request by vagetablechicken**

### CODE REVIEW

File: docs/zh/deploy/install_deploy.md [hunk 1]
```diff
@@ -83,6 +83,16 @@ Wed Aug 22 16:33:50 CST 2018
 ```
 请确保时间是正确的
 
+## 网络白名单
+
+OpenMLDB集群的服务组件之间需要保证网络连通。
+
+客户端到OpenMLDB集群则分为两种情况：
+
+- client（CLI/各SDK）连接OpenMLDB集群，除了与zookeeper的连通，还需要保证和nameserver/tabletserver/taskmanager的连通，因为client操作可能会直连某个server。
```
Comment (lumianph) [inline docs/zh/deploy/install_deploy.md]: let's use the formal and consistent names: ZooKeeper, TaskManager, APIServer, IP, etc.

File: docs/zh/deploy/install_deploy.md [hunk 2]
```diff
@@ -83,6 +83,16 @@ Wed Aug 22 16:33:50 CST 2018
...
+
+- 如果服务仅使用apiserver作为代理，那么客户端只需保证能访问apiserver端口。（apiserver作为服务组件，必然保证和OpenMLDB集群其他组件的连通性。）
+
 ## 预备测试
 
 由于linux平台的多样性，发布包可能在你的机器上不兼容，请先通过简单的运行测试。
```
Comment (lumianph) [inline docs/zh/deploy/install_deploy.md]: Linux

**Review by lumianph — APPROVED** [review]

File: docs/zh/deploy/install_deploy.md [hunk 3]
```diff
@@ -2,16 +2,17 @@
...
   * 目前仅支持 x86 架构，暂不支持例如 ARM 等架构。
   * 核数推荐不少于 4 核，如果 Linux 环境下 CPU 不支持 AVX2 指令集，需要从源码重新编译部署包。
 
-* 运行环境：zookeeper和taskmanager部署需要java runtime environment。其他组件无要求。
+* 运行环境：ZooKeeper和TaskManager部署需要java runtime environment。其他组件无要求。
 
 ## 部署包准备
-本说明文档中默认使用预编译好的 OpenMLDB 部署包（[Linux](https://github.com/4paradigm/OpenMLDB/releases/download/v0.7.0/openmldb-0.7.0-linux.tar.gz), [macOS](https://github.com/4paradigm/OpenMLDB/releases/download/v0.7.0/openmldb-0.7.0-darwin.tar.gz)），所支持的操作系统要求为：CentOS 7, Ubuntu 20.04, macOS >= 10.15。如果用户期望自己编译（如做 OpenMLDB 源代码开发，操作系统或者 CPU 架构不在预编译部署包的支持列表内等原因），用户可以选择在 docker 容器内编译使用或者从源码编译，具体请参照我们的[编译文档](compile.md)。
+
+本说明文档中默认使用预编译好的 OpenMLDB 部署包（[Linux](https://github.com/4paradigm/OpenMLDB/releases/download/v0.7.0/openmldb-0.7.0-linux.tar.gz), [macOS](https://github.com/4paradigm/OpenMLDB/releases/download/v0.7.0/openmldb-0.7.0-darwin.tar.gz)），所支持的操作系统要求为：CentOS 7, Ubuntu 20.04, macOS 12。如果用户期望自己编译（如做 OpenMLDB 源代码开发，操作系统或者 CPU 架构不在预编译部署包的支持列表内等原因），用户可以选择在 docker 容器内编译使用或者从源码编译，具体请参照我们的[编译文档](compile.md)。
```
Comment (dl239) [inline docs/zh/deploy/install_deploy.md]: 0.7.1

File: docs/zh/deploy/install_deploy.md [hunk 4]
```diff
@@ -83,17 +84,27 @@ Wed Aug 22 16:33:50 CST 2018
...
+由于Linux平台的多样性，发布包可能在你的机器上不兼容，请先通过简单的运行测试。
 ```
 wget https://github.com/4paradigm/OpenMLDB/releases/download/v0.7.0/openmldb-0.7.0-linux.tar.gz
 tar -zxvf openmldb-0.7.0-linux.tar.gz
 ./openmldb-0.7.0-linux/bin/openmldb --version
 ```
 结果应显示该程序的版本号，类似
 ```
-openmldb version 0.6.3-xxxx
+openmldb version 0.7.0-xxxx
```
Comment (dl239) [inline docs/zh/deploy/install_deploy.md:107]: 0.7.1

File: docs/zh/deploy/install_deploy.md [hunk 5]
```diff
@@ -116,31 +127,31 @@ cd openmldb-0.7.0-linux
...
-#### 3 配置apiserver：conf/standalone_apiserver.flags
+#### 3 配置APIServer：conf/standalone_apiserver.flags
 
 APIServer负责接收http请求，转发给OpenMLDB并返回结果。它是无状态的，而且并不是OpenMLDB必须部署的组件。
 运行前需确保OpenMLDB其他服务已经启动，否则APIServer将初始化失败并退出进程。
 
-* 修改endpoint。endpoint是用冒号分隔的部署机器ip/域名和端口号
-* 修改nameserver为nameserver的地址
+* 修改endpoint。endpoint是用冒号分隔的部署机器IP/域名和端口号
+* 修改NameServer为NameServer的地址
```
Comment (dl239) [inline docs/zh/deploy/install_deploy.md]: 第一个nameserver为配置名，不需要驼峰。上面的tablet也是

**Review by dl239 — APPROVED** [review]

**Pull request merged**